src/dev/impl/DevToys/Views/Tools/Converters/NumberBaseConverter/AdvancedNumberBaseConverterControl.xaml

```diff
@@ -0,0 +1,73 @@
+﻿<UserControl
+    x:Class="DevToys.Views.Tools.Converters.NumberBaseConverter.AdvancedNumberBaseConverterControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevToys.UI.Controls"
+    xmlns:DevToys="using:DevToys"
+    xmlns:converters="using:DevToys.UI.Converters"
+    mc:Ignorable="d"
+    d:DesignHeight="400"
+    d:DesignWidth="600">
+
+    <UserControl.Resources>
+        <converters:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
+    </UserControl.Resources>
+    
+    <StackPanel Margin="0,0,0,16" Spacing="12">
+        <StackPanel.ChildrenTransitions>
+            <TransitionCollection>
+                <EntranceThemeTransition IsStaggeringEnabled="True" FromVerticalOffset="50"/>
+            </TransitionCollection>
+        </StackPanel.ChildrenTransitions>
+
+        <StackPanel Spacing="4" >
+            <controls:ExpandableSettingControl
+                x:Name="InputDictionaryValue"
+                Grid.Row="2"
+                Title="{x:Bind ViewModel.Strings.InputDictionaryLabel}">
+                <controls:ExpandableSettingControl.Icon>
+                    <FontIcon />
+                </controls:ExpandableSettingControl.Icon>
+                <ComboBox
+                    SelectedItem="{x:Bind ViewModel.InputNumberFormat, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                    ItemsSource="{x:Bind ViewModel.BaseNumbers}">
+                </ComboBox>
+            </controls:ExpandableSettingControl>
+            
+            <TextBox Visibility="{x:Bind ViewModel.UseInputCustomDictionary, Mode=OneWay, Converter={StaticResource BooleanToVisibilityConverter}}" 
+                     Text="{x:Bind ViewModel.InputCustomDictionary, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>
+            
+            <controls:CustomTextBox
+                Header="{x:Bind ViewModel.Strings.DecimalLabel}"
```

Comment:
Interestingly, the header doesn't show up on my side:

![image](https://user-images.githubusercontent.com/3747805/158031775-63085cb5-2186-4e7e-849c-e5e0be389792.png)

Reply:
Yes, that's really interesting. I haven't noticed it has a wrong label as I never saw it when testing the app. This issue also happens in the output textbox...

Reply:
It seems like when the UserControl (`AdvancedNumberBaseConverterControl` in this case) is not visible the values for `Header` in `CustomTextBox` are not correctly bound, so it is set to null. I cannot tell why is this happening.